TodoList/TodoList/appsettings.json

```
@@ -9,5 +9,10 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "Jwt": {
+    "Key": "DhftOS5uphK3vmCJQrexST1RsyjZBjXW",
```

```suggestion
    "Key": "DhftOS5uphK3vmCJQrexST1RsyjZBjXW",
```
e cần nhắc dùng user secret nhé. thường thì cái này k đấy lên github đâu

---

https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-7.0&tabs=windows